core/drivers/crypto/se050/crypto.mk

```diff
@@ -0,0 +1,67 @@
+ifeq ($(CFG_NXP_SE05X), y)
+# Enable the crypto driver
+CFG_CRYPTO_DRIVER = y
```

Comment:
ok

Reply:
@etienne-lms all these crypto.mk changes that you are asking dont take into consideration the imx default platform config which assumes that everything will be provided by CAAM (if fitted)....so I will have to work around all the issues in imx-platform. 

please confirm you want this changes instead of what I have currently delivered (which dont need changes to core/arch/arm/plat-imx/crypto_conf.mk )....I'd hate to modify plat-imx to then be asked to undo those changes.

Reply:
So `CFG_NXP_SE05X=y` really mandates `CFG_CRYPTO_DRIVER=y`, right?

Reply:
yes, 100%.